samples/webview/webview.cpp

```diff
@@ -383,13 +383,21 @@ WebFrame::WebFrame(const wxString& url) :
         wxLogMessage("Edge backend not available");
     }
 #endif
-    m_browser = wxWebView::New(this, wxID_ANY, url, wxDefaultPosition, wxDefaultSize, backend);
+    m_browser = wxWebView::New(backend);
+#ifdef __WXMAC__
+    // With WKWebView handlers need to be registered before creation
```

Comment:
Does this work with the other backends too or do we have to do it differently depending on the platform?

In any case, it would be nice to mention this in `RegisterHandler()` documentation.

Reply:
Unfortunately with other backends it's the opposite  where the `RegisterHandler()` call has to be made later. I'll add that to the documentation.